staging/src/k8s.io/apimachinery/pkg/runtime/error.go

```diff
@@ -71,11 +72,11 @@ func (k *notRegisteredErr) Error() string {
 // IsNotRegisteredError returns true if the error indicates the provided
 // object or input data is not registered.
 func IsNotRegisteredError(err error) bool {
+	var nerr *notRegisteredErr
 	if err == nil {
```

Comment:
We can drop this if condition since `errors.As` handles nil

Reply:
thx, updated